src/dispatch/incident/flows.py

```diff
@@ -772,16 +765,44 @@ def incident_closed_flow(incident_id: int, command: Optional[dict] = None, db_se
             if storage_plugin:
                 storage_plugin.instance.open(incident.storage.resource_id)
 
-    # we delete the tactical and notification groups
-    delete_participant_groups(incident, db_session)
-
-    # we delete the conference
-    delete_conference(incident, db_session)
-
     db_session.add(incident)
     db_session.commit()
 
 
+def status_flow_dispatcher(
+    incident: Incident,
+    current_status: IncidentStatus,
+    previous_status: IncidentStatus,
+    db_session=SessionLocal,
+):
+    """Runs the correct flows depending on a incident's current and previous status."""
+
+    # we have a currently active incident
+    if current_status == IncidentStatus.active:
+        # re-activate incident
+        if previous_status == IncidentStatus.closed:
+            incident_active_status_flow(incident=incident, db_session=db_session)
+        elif previous_status == IncidentStatus.stable:
+            incident_active_status_flow(incident=incident, db_session=db_session)
```

Comment:
Do we need the `if / elif` here if we always call `incident_active_status_flow()`?
```suggestion
        incident_active_status_flow(incident=incident, db_session=db_session)
```